l10n_br_hr_contract/models/hr_contract_notice_termination.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016  Daniel Sadamo - KMEE Informática
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields
+
+
+class HrContractNoticeTermination(models.Model):
+    _name = 'hr.contract.notice.termination'
+
+    name = fields.Char(string='Notice of termination type')
```

Comment:
This class must have at least one mandatory field to prevent an empty record creation